src/utils/todo/task-filing/button-loader.ts

```diff
@@ -108,14 +108,14 @@ const showDeleteOnly = (todoItem: TodoItemI): boolean => {
   const business = useBcrosBusiness()
   const { isStaffAccount } = useBcrosAccount()
   const filingType = todoItem.name
-  if (filingType === FilingTypes.ALTERATION || filingType === FilingTypes.DISSOLUTION) {
-    // Alteration filing draft and Dissolution filing draft can only be deleted
-    return true
-  } else if (filingType === FilingTypes.SPECIAL_RESOLUTION) {
-    // if a business is not in good standing, non-staff role can only delete the Special Resolution draft
-    return business && !business.currentBusiness.goodStanding && !isStaffAccount
-  } else {
-    return false
+
+  switch (filingType) {
+    case FilingTypes.ALTERATION:
+    case FilingTypes.DISSOLUTION:
```

Comment:
Just a question, so for delete button show we treat both alteration and dissolution as if it is special resolution ?

Reply:
Yes. The switch statement is copied from the old codebase. I misunderstood the logic before.